frontend/apps/filemanager/filemanager.lua

```diff
@@ -455,6 +461,10 @@ function FileManager:init()
     FileManager.instance = self
 end
 
+function FileManager:onKeyToggleWifi()
+    return self.networklistener:onToggleWifi()
+end
```

Comment:
I understand this still needs to be removed, pinging @yparitcher about what else might need to be adjusted regarding wifi since this suddenly became unmergable just as I tought it was ready. :-/
```suggestion

```